specification/containerservice/resource-manager/readme.md

```diff
@@ -44,8 +44,12 @@ These settings apply only when `--tag=package-2019-04` is specified on the comma
 
 ```yaml $(tag) == 'package-2019-04'
 input-file:
+  - Microsoft.ContainerService/preview/2018-09-30-preview/openShiftManagedClusters.json
```

Comment:
Are these connected to the latest version changes ?

Reply:
These are only included in the package. It is a separate resource and API version